react/features/facial-recognition/functions.js

```diff
@@ -5,6 +5,24 @@ import { extractFqnFromPath } from '../dynamic-branding';
 import { SET_TIMEOUT } from './constants';
 import logger from './logger';
 
+/**
+ * Retrieves app base url.
+ *
+ * @returns {string}
+ */
+export function getAppBaseUrl() {
```

Comment:
I see that we have `getBaseUrl` in react/features/base/util/helpers.js. Can't we reuse it?

Reply:
yep, that's right, didn't know about it